Kitodo/src/main/webapp/WEB-INF/templates/includes/userEdit/userEditPopup.xhtml

```diff
@@ -40,6 +40,7 @@
                                            action="#{BenutzerverwaltungForm.addToGroup}"
                                            title="add to user group"
                                            update="editForm">
+                                           oncomplete="jQuery('#editForm\\:saveButtonToggler').click()">
```

Comment:
You are closing a non open tag, this will cause an error or the functionality is not working.